src/types.ts

```diff
@@ -0,0 +1,16 @@
+export interface TimeUnits {
+  magnitude: number;
+  name: string;
+}
+
+export interface PlaybackState {
+  currentTime: number;
+  isPlaying: boolean;
+}
+
+export interface PlaybackData {
+  timeStep: number;
```

Comment:
Is timestep always guaranteed to be an int? I'm thinking about the edge case where it's a float value, and due to float arithmetic we end up with slight imprecision on the current time.

Reply:
No it is not required to be an int

Reply:
The example trajectory in this repo currently has a timeStep of 0.5 so if you advance forward frame by frame you'll see that's how it increments. I think the `compareTimes` util in the viewer repo is meant to handle a lof o those edge cases, but I'll try to think of how it might trip us up...